basics/timelock-contract/src/TimelockContractState.cs

```diff
@@ -0,0 +1,14 @@
+using AElf.Sdk.CSharp.State;
+using AElf.Types;
+
+namespace AElf.Contracts.Timelock
+{
+    public partial class TimelockContractState : ContractState 
+    {
+        public BoolState Initialized { get; set; }
+        public SingletonState<Address> Admin { get; set; }
+        public SingletonState<Address> PendingAdmin { get; set; }
+        public SingletonState<long> Delay { get; set; }
+        public MappedState<Hash, Address> TransactionQueue { get; set; }
```

Comment:
Previously, because the old ExecuteTransaction method used tokenContract.transferFrom method, and It need "from" field at that time. I put this field to this MappedState.